components/tls/tls-core/Cargo.toml

```diff
@@ -1,5 +1,10 @@
 [package]
 name = "tlsn-tls-core"
+authors = ["TLSNotary Team"]
+description = "Cryptography for the TLSNotary tls client"
```

Comment:
cryptography sound more like "the science of cryptography" to me.
"Cryptographic operations for the TLSNotary TLS client"